### PR TITLE
docs(website): updated the docker compatibility section

### PR DESCRIPTION
**Pull request by shipsing**
### What does this PR do?
Look at the [migrating from Docker](https://podman-desktop.io/docs/migrating-from-docker) section in entirety to remove the sections that are not needed anymore (because UI implementation provides that capability) and edit the sections for clarity. 

### Screenshot / video of UI
PR preview: https://673aca6ae85ee2beaa1206f3--podman-desktop-pr.netlify.app/docs/migrating-from-docker

### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop/issues/9138

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Comment by shipsing** [comment]
Once the Docker Compatibility feature is fully implemented in UI, we would need to remove the following sections that cover the manual approach:

- https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman
- https://podman-desktop.io/docs/migrating-from-docker/verifying-your-tools-are-using-podman
- https://podman-desktop.io/docs/migrating-from-docker/using-podman-mac-helper 

Edit [this section](https://podman-desktop.io/docs/migrating-from-docker/using-the-docker_host-environment-variable) to highlight the content is applicable only to Windows and Linux platforms. Because with the Third-Party Docker Tool compatibility settings avaialable on macOS, you can use the third-party tools without following the manual approach mentioned in that section.

More concised and focused approach to quickly set up Docker compatibility would be to first [import your saved containers using CLI](https://podman-desktop.io/docs/migrating-from-docker/importing-saved-containers) and then access the [Docker Compatibility page](https://podman-desktop.io/docs/migrating-from-docker/managing-docker-compatibility) for configuring the settings. 

Also, need to add a screenshot in the [Docker Compatibility](https://podman-desktop.io/docs/migrating-from-docker/managing-docker-compatibility) section.

**Comment by cdrage** [comment]
> Once the Docker Compatibility feature is fully implemented in UI, we would not need to remove the following sections that cover the manual approach:
> 
> * https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman
> * https://podman-desktop.io/docs/migrating-from-docker/verifying-your-tools-are-using-podman
> * [https://podman-desktop.io/docs/migrating-from-docker/using-podman-mac-helper ](https://podman-desktop.io/docs/migrating-from-docker/using-podman-mac-helper%C2%A0)
> 
> Edit [this section](https://podman-desktop.io/docs/migrating-from-docker/using-the-docker_host-environment-variable) to highlight the content is applicable only to Windows and Linux platforms. Because with the Third-Party Docker Tool compatibility settings avaialable on macOS, you can use the third-party tools without following the manual approach mentioned in that section.
> 
> More concised and focused approach to quickly set up Docker compatibility would be to first [import your saved containers using CLI](https://podman-desktop.io/docs/migrating-from-docker/importing-saved-containers) and then access the [Docker Compatibility page](https://podman-desktop.io/docs/migrating-from-docker/managing-docker-compatibility) for configuring the settings. 
> 
> Also, need to add a screenshot in the [Docker Compatibility](https://podman-desktop.io/docs/migrating-from-docker/managing-docker-compatibility) section.

Should this PR be in draft then until the PR for compatibility section is in?

Otherwise, the docs LGTM (as it's just removing or moving paragraphs around anyways).

**Review by benoitf — APPROVED** [review]
- if we can have redirects for page being removed (for search engines)
- need to keep DOCKER_HOST section for macOS (people can do that)
- further enhancement: replace DOCKER_HOST usage by `docker context` command, where we can add a podman context and then switch to this context using `docker context use podman` and then use the docker CLI against a remote podman engine

**Review by Firewall — APPROVED** [review]

**Comment by shipsing** [comment]
@benoitf, I have incorporated your suggestions. Could you please verify the PR? 
Also, can we merge the PR now or do we have to wait till 4.15 is out?

File: website/docs/migrating-from-docker/using-the-docker_host-environment-variable.md [hunk 1]
```diff
@@ -9,14 +9,22 @@ tags: [migrating-from-docker]
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Using the `DOCKER_HOST` environment variable
+# Using the `DOCKER_HOST` environment variable
```
Comment (benoitf) [inline website/docs/migrating-from-docker/using-the-docker_host-environment-variable.md]:
```suggestion
# Using the `DOCKER_HOST` environment variable
```

File: website/docs/migrating-from-docker/using-the-docker_host-environment-variable.md [hunk 2]
```diff
@@ -9,14 +9,22 @@ tags: [migrating-from-docker]
...
 
 - Continue using familiar Docker commands.
 - Take advantage of the benefits of Podman.
 - Your tools, such as [Gradle](https://gradle.org/) or [Testcontainers](https://www.testcontainers.com/), communicate with Podman without reconfiguration.
 
+Using the `DOCKER_HOST` environment variable, you can connect your Docker CLI to the Podman socket.
+
+:::note
+
+Alternatively, you can add a `podman` context using the `docker context create` command. Then, you can run the `docker context use podman` command to switch to that context. This way you can use your Docker CLI to run your tasks on a remote Podman engine. When you run the `docker ps` command, it queries the Podman socket specified in the current context.
```
Comment (benoitf) [inline website/docs/migrating-from-docker/using-the-docker_host-environment-variable.md]: suggestion: maybe tell that the value of the context should follow the pattern:

`docker context create podman --docker "host=unix://$HOME.local/share/containers/podman/machine/podman.sock"`

(for example on macOS)
so `host=unix://<$DOCKER_HOST value>`

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
> Also, can we merge the PR now or do we have to wait till 4.15 is out?

I think it's ok having that merged now and not waiting until 4.15

**Pull request merged**